src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -207,6 +214,16 @@ export class ChartContainer extends React.Component<IChartContainerProps, {}> {
                     this.props.onToggleLogScaleY(this.props.chartMeta);
                 }
             }),
+            onToggleBoxPlot: action(() => {
+                if (this.props.onToggleBoxPlot) {
```

Comment:
fyi, you can now use "optional chaining" to shorten this.  ```this.props.onToggleBoxPlot?.()``` will not throw an error if the property is undefined